Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.MixedReality.Toolkit.Examples.Demos.Gltf.Editor
     [CustomEditor(typeof(TestGltfLoading))]
     public class TestGltfLoadingEditor : UnityEditor.Editor
     {
-        private static readonly string GLTFModelsPath = $"Demos{Path.DirectorySeparatorChar}Gltf{Path.DirectorySeparatorChar}Models";
+        private static readonly string GLTFModelsPath = "Common/Gltf/Models";
```

Comment:
To keep the original formatting?:

```suggestion
        private static readonly string GLTFModelsPath = $"Common{Path.DirectorySeparatorChar}Gltf{Path.DirectorySeparatorChar}Models";
```